test-e2e/cip-auditor/cip-auditor-e2e.go

```diff
@@ -319,10 +319,10 @@ func getWorkspaceStatus(repoRoot string) map[string]string {
 	}
 
 	stdout := std.Output()
-	for idx, line := range strings.Split(strings.TrimSuffix(stdout, "\n"), "\n") {
+	for _, line := range strings.Split(strings.TrimSuffix(stdout, "\n"), "\n") {
 		words := strings.Split(line, " ")
 		if len(words) != 2 {
-			klog.Fatalf("Unexpected key value pair in line: %d!\n", idx)
+			klog.Fatalf("Error: Unexpected key value pair when parsing workspace_status.sh.\nLine: \"%s\"", line)
```

Comment:
Not sure if the leading "Error: " prefix is required (unless we already do this elsewhere, can you check?).

Can you also include the line number (`idx`) in the log output as well?

Also, you can drop the embedded newlines. Also you should use `%q` instead. See https://golang.org/pkg/fmt/

Reply:
Fatal errors are not consistent. However, [here](https://github.com/kubernetes-sigs/k8s-container-image-promoter/blob/master/test-e2e/cip-auditor/cip-auditor-e2e.go#L127-#L148) in cip-auditor-e2e.go there's a pattern of using the `error` keyword at the beginning of the log. This makes it easier to find errors (with ctrl-f search) inside huge logfiles.

Reply:
FYI It might be worth going with `ERROR:` because prow highlights these in the log
https://github.com/kubernetes/test-infra/blob/bb8a234a43a9824d8f7518b51321d826245926ff/config/prow/config.yaml#L62